test/stdlib/MultilineString.swift

```diff
@@ -0,0 +1,56 @@
+// RUN: %target-run-stdlib-swift | %FileCheck %s
+// REQUIRES: executable_test
+
+// RUN: %target-build-swift %s 2> %t.warnings.txt
+// RUN: %FileCheck -check-prefix=CHECK-WARNINGS %s < %t.warnings.txt
+
+import Swift
+
+// ===---------- Multiline --------===
+
+func delimit(_ str: String) -> String {
+  return "<\(str)>"
+}
+
+// CHECK: -1-
+print("-1-")
+// CHECK-NEXT: {{^}}<Six
+// CHECK-NEXT: {{^}}""Zeta"">
+print(delimit("""
+    Six
+    ""Zeta""
+    """
+))
+
+// CHECK: -2-
+print("-2-")
+// CHECK-NEXT: {{^}}<  Eight
+// CHECK-NEXT: {{^}}Iota>
+print(delimit("""
+    Eight
+  Iota
+  """
+))
+
+// CHECK: -3-
+print("-3-")
+// CHECK-NEXT: {{^}}<  Nine
+// CHECK-NEXT: {{^}}  Kappa>
+print(delimit("""
+    Nine
+    Kappa
+  """
+))
+
+// CHECK: -4-
+print("-4-")
+// CHECK-NEXT: {{^}}<    Ten
+// CHECK-NEXT: {{^}}    Lambda>
+print(delimit("""
+    Ten
+    Lambda
+"""))
+
+// ===---------- Done --------===
+// CHECK-NEXT: Done.
+print("Done.")
```

Comment:
@kubamracek, the limitation with respect to normalising arbitrary combinations of end of lines within a literal are due to how the code operates using a find/replace. It would require a bit of a rewrite to cater to this edge case and would be headed into diminishing returns IMHO. Normally a source file would not mix line endings inside a literal.